src/sentry/static/sentry/app/views/organizationProjectsDashboard/index.jsx

```diff
@@ -111,15 +110,16 @@ class Dashboard extends React.Component {
 }
 
 const OrganizationDashboard = createReactClass({
```

Comment:
Why not remove the create react class while you're here?

Reply:
Just keeping scope low, `createReactClass` can be codemodded later